### ISSUE TITLE
Blightfall world doesn't load

### ISSUE THREAD

**Issue by Varannn** [talonos2/BlightfallTracker#239]
Hiya, when I try to enter the world, it stucks at "Downloading terrain" or something. Got a 45 MB logfile. Can I upload here?

**Comment by talonos2** [comment]
I'd rather you open it and check to see if there's a stacktrace repeated
over a thousand times, and just give me that stack trace. No need to clog
github's servers. :)

**Comment by Varannn** [comment]
Ok tbh I have no clue about this stuff. Somehow everything seems to be repeated. Somehow the world is corrupted. Please have a look. Maybe the corruption is removable and preventable in the future:
https://ufile.io/ndmeh

**Comment by talonos2** [comment]
Right. I figured everything would be repeated. That's why I asked you to
give me one "copy" of the repeated section and send it to me. But whatever.
I've download the log.

Try this:

Go into the config folder. Locate the file "Forge.cfg"; there is a line for
removing erroring tile entities. turn this from false to true. Restart the
server and load the chunks the crash is happening at. Afterwards turn the
config back to false and restart the server.

**Comment by Varannn** [comment]
I turned it already to "true" but I still can't enter the world. There is no difference, it stucks at "Downloading terrain"
